src/main/resources/application.properties

```diff
@@ -6,16 +6,21 @@ spring.output.ansi.enabled=ALWAYS
 #logging.level.root=DEBUG
 
 ## Database
-# spring.datasource.dbname=app_fake_data
+#spring.datasource.dbname=app_fake_data
+
 spring.datasource.url=jdbc:mysql://localhost:3306/sulaahDB
 spring.datasource.username=root
 spring.datasource.password=samplePassword1
 spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
-spring.jpa.database-platform=org.hibernate.dialect.MySQLDialect
-# spring.datasource.initialization-mode=never
+
+#spring.jpa.database-platform=org.hibernate.dialect.MySQLDialect
```

Comment:
I dont think this line should be commented

Reply:
`spring.jpa.database-platform=org.hibernate.dialect.MySQLDialect`
this one

Reply:
Uncomment it